donkeycar/parts/object_detector/stop_sign_detector.py

```diff
@@ -23,7 +23,7 @@ def download_file(self, url, filename):
         if not os.path.isfile(filename):
             urllib.request.urlretrieve(url, filename)
 
-    def __init__(self, min_score, show_bounding_box, debug=False):
+    def __init__(self, min_score, show_bounding_box, max_reverse_count, debug=False):  

```

Comment:
Can you provide a default (probably 0 here), so we retain the previous behaviour if the variable is not set?